src/smart-components/App/DesktopLayout.tsx

```diff
@@ -104,6 +123,20 @@ export const DesktopLayout: React.FC<DesktopLayoutProps> = (
           />
         </div>
       )}
+      {showThread && (
+        <Thread
+          className="sendbird-app__thread"
+          channelUrl={currentChannel?.url || ''}
```

Comment:
Do we really need `channelUrl={currentChannel?.url || ''} ` or is `channelUrl={currentChannel?.url}` enough?